packages/images/lib/compare.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
```

Comment:
There is a pending PR https://github.com/appium/appium-base-driver/pull/449 
I'm not quite when/if @khaong has time to work on it. Otherwise there going to be a conflict

Reply:
If this lands in the Appium 1.x branch before we cut over to 2.x, I'm happy to port it to the plugin myself. Otherwise, probably best for @khaong to close on basedriver and reopen against the plugin repo.

Reply:
Hi @jlipps , what's your timeline? I'll probably have something ready to review by this weekend.

Reply:
there's no particular hurry. we're not going to cut over to 2.x immediately. so get your patch into the current version of basedriver, and then i can port it over to this plugin, maybe asking for some of your help along the way.

Reply:
👍  cool, thanks